### DISCUSSION TITLE
Hiding R messages

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#3451] [Q&A] [answered]
Hi,
I have very simple code, to render CSV file.

In the rendered HTML page I have a lot of R messages.
Is it normal to have them? How to hide them?

Thank you

![image](https://user-images.githubusercontent.com/30607/203863526-5f3ae627-47cf-42d0-866e-1d568225a83c.png)

~~~~
---
title: "Lorem"
format: html
execute:
  daemon: false
  freeze: auto
---

```{r}
#| echo: false
library(tidyverse)
library(lubridate)
library(knitr)
```


```{r}
#| echo: false

t = read_csv("tmp.csv")
library(knitr)
kable(head(t))
```


```{r}

#| echo: false

t = read_csv("tmp.csv")

rmarkdown::paged_table(t)
```
~~~~

**Comment by baptiste** [comment]
You can hide messages with

```
---
title: "Lorem"
knitr: 
  opts_chunk:
    message: false
---
```

**Reply by aborruso** [reply]
> You can hide messages with

Thank you.

I have still this

![image](https://user-images.githubusercontent.com/30607/203872135-db79fbfd-4537-4878-909a-5ccbf103ae9d.png)

**Comment by baptiste** [comment, marked as answer]
that's a warning, so `warning: false` should hide it

**Reply by aborruso** [reply]
thank you
